package.json

```diff
@@ -34,6 +34,7 @@
     "@react-native-community/eslint-config": "^0.0.3",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^25.2.6",
+    "babel-plugin-macros": "^2.8.0",
```

Comment:
Moving this line seems unrelated?

Reply:
Correct, this was autofixed when running yarn

Reply:
As for the fix, I updated the CLI following the steps here: https://github.com/react-native-community/cli#updating-the-cli